encode_test.go

```diff
@@ -425,3 +425,19 @@ func TestParseRawContentTextOptionFalse(t *testing.T) {
 	}
 	test.DiffGolden(t, b.Bytes(), "testdata", "encode", "parser-raw-content-text-option-false.raw.golden")
 }
+
+// TestRawContentUTF8Headers test if not encoded UTF8 headers are not edited with the rawContent parser option.
```

Comment:
Would this be more correct?

```suggestion
// TestRawContentUTF8Headers verifies plain-text headers are unmodified with the rawContent parser option.
```